lib/3scale_toolbox/commands/copy_command/copy_service.rb

```diff
@@ -40,7 +40,8 @@ def run
             Tasks::CopyApplicationPlansTask.new(context),
             Tasks::CopyLimitsTask.new(context),
             Tasks::DestroyMappingRulesTask.new(context),
-            Tasks::CopyMappingRulesTask.new(context)
+            Tasks::CopyMappingRulesTask.new(context),
+            Tasks::CopyPoliciesTask.new(context)
```

Comment:
better to have a trailing comma there IMO, to reduce unnecessary changes when adding items

Reply:
:+1: